src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
     public String getRandomColor() {
-        return null;
+        return String.valueOf(Colors.values()[new Random().nextInt(Colors.values().length)]);
```

Comment:
Let's split it up a bit first because it is not readable at the moment.
Then we make random a class field to avoid creating a new one each time
After that, we can add a variable for `Color.values()` call result to avoid multiple calls